src/licensedcode/data/rules/ibm_developerworks.yml

```diff
@@ -0,0 +1,2 @@
+licenses:
+    - ibm-developerworks-community-download

```

Comment:
Dennis suggested ` ibm-developerworks-community-download` is a good match, please check his comments in #174